src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -15,6 +15,10 @@
             <argument type="service" id="cache.app" />
         </service>
 
+        <service id="cache.assets" parent="cache.app" public="false">
```

Comment:
`cache.app` means the cache can be shared when there are several front servers but it also means the cache won't be cleared on redeploy. I think this should be cleared on redeploy, isn't it? Then, there are two ways to do so: either extend `cache.system` - this will benefit from a local cache and will be cleared on redeploy. Or add a cache warmer to clear the cache when `cache:clear` runs (it could also really warmup the cache btw).

I don't know what's best. WDYT?

Reply:
For simplicity, I get the `cache.system` option.